### PR TITLE
FIX: Fixed bug which prevented to skip healing on nastran import

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: src/ansys/aedt/core/visualization/advanced/misc.py [hunk 1]
```diff
@@ -557,32 +557,27 @@ def _write_stl(nas_to_dict, decimation, working_directory, enable_planar_merge=T
...
+            normal = GeometryOperators.normalize_vector(n)
             f.write(f" facet normal {normal[0]} {normal[1]} {normal[2]}\n")
             f.write("  outer loop\n")
             f.write(f"   vertex {points[0][0]} {points[0][1]} {points[0][2]}\n")
             f.write(f"   vertex {points[1][0]} {points[1][1]} {points[1][2]}\n")
             f.write(f"   vertex {points[2][0]} {points[2][1]} {points[2][2]}\n")
             f.write("  endloop\n")
             f.write(" endfacet\n")
+        except Exception:
+            pass
```
Comment (SMoraisAnsys) [inline src/ansys/aedt/core/visualization/advanced/misc.py:580]:
```suggestion
        except Exception:  # pragma: no cover
            logger.debug("Failed to normalize vector.")
```

File: src/ansys/aedt/core/visualization/advanced/misc.py [hunk 2]
```diff
@@ -557,32 +557,27 @@ def _write_stl(nas_to_dict, decimation, working_directory, enable_planar_merge=T
...
-        elif cv2[0] == cv1[0] == 0.0 and cv2[2] == cv1[2] == 0.0:
-            n = [0, 1, 0]  # pragma: no cover
-        elif cv2[1] == cv1[1] == 0.0 and cv2[2] == cv1[2] == 0.0:
-            n = [1, 0, 0]  # pragma: no cover
-        else:
+        fc = points[0]
+        v1 = points[1]
+        v2 = points[2]
+        cv1 = GeometryOperators.v_points(fc, v2)
+        cv2 = GeometryOperators.v_points(fc, v1)
```
Comment (SMoraisAnsys) [inline src/ansys/aedt/core/visualization/advanced/misc.py:567]:
```suggestion
        cv1 = GeometryOperators.v_points(fc, v1)
        cv2 = GeometryOperators.v_points(fc, v2)
```
Any reason to changes the name values ? I expected cv1 to be the distance between v1 and the center (same for cv2)

File: src/ansys/aedt/core/visualization/advanced/misc.py [hunk 3]
```diff
@@ -557,32 +557,27 @@ def _write_stl(nas_to_dict, decimation, working_directory, enable_planar_merge=T
     def _write_solid_stl(triangle, pp):
         try:
             # points = [nas_to_dict["Points"][id] for id in triangle]
-            points = [pp[i] for i in triangle]
+            points = [pp[i] for i in triangle[::-1]]
```
Comment (SMoraisAnsys) [inline src/ansys/aedt/core/visualization/advanced/misc.py:560]: Any reason for using `[::-1]` ?

File: src/ansys/aedt/core/modeler/cad/polylines.py [hunk 4]
```diff
@@ -341,14 +341,15 @@ def __init__(
 
             self._is_closed = close_surface
             self._is_covered = cover_surface
-
+            self._solve_inside = False
```
Comment (SMoraisAnsys) [inline src/ansys/aedt/core/modeler/cad/polylines.py:344]: Why are those changes required ?

**Review by SMoraisAnsys — DISMISSED** [review]
LGTM I left some questions and minor comments.

File: src/ansys/aedt/core/visualization/advanced/misc.py (hunk 2 above)
Comment (maxcapodi78) [reply src/ansys/aedt/core/visualization/advanced/misc.py:567]: it's the sign of the vector that will be impacted

File: src/ansys/aedt/core/visualization/advanced/misc.py (hunk 3 above)
Comment (maxcapodi78) [reply src/ansys/aedt/core/visualization/advanced/misc.py:560]: order is important for stl creation

File: src/ansys/aedt/core/modeler/cad/polylines.py (hunk 4 above)
Comment (maxcapodi78) [reply src/ansys/aedt/core/modeler/cad/polylines.py:344]: to prevent AEDT to fail in creating Polyline in 2D Modeler

**Review by Samuelopez-ansys — APPROVED** [review]
